CMakeLists.txt

```diff
@@ -158,6 +158,8 @@ option(EXECUTORCH_BUILD_PYBIND "Build the Python Bindings" OFF)
 
 option(EXECUTORCH_BUILD_QNN "Build the Qualcomm backend" OFF)
 
+option(REGISTER_OPTIMIZED_OPS "Build the optimized kernels" ON)
```

Comment:
All options must begin with `EXECUTORCH_` to avoid conflicts when we're included as a submodule; we've drifted away from that as more and more people have hacked on the cmake files, but we need to get back there.

Reply:
Good point. I was matching the convention from REGISTER_QUANTIZED_OPS, but I'll update both to use the EXECUTORCH_ prefix.